statsmodels/genmod/generalized_linear_model.py

```diff
@@ -963,9 +985,9 @@ def _fit_irls(self, start_params=None, maxiter=100, tol=1e-8,
 
         # first guess on the deviance is assumed to be scaled by 1.
         # params are none to start, so they line up with the deviance
-        history = dict(params=[None, start_params], deviance=[np.inf, dev])
+        history = dict(params=[np.inf, start_params], deviance=[np.inf, dev])
```

Comment:
I added `inf` because `allclose` in the `_check_convergence` function didn't like comparing `None` to an array.